### PR TITLE
docs - add shift-left to main readme, flush out c7n-left readme

### PR DESCRIPTION
**Pull request by kapilt**
add c7n-left to main readme for discoverability

add traverse filter docs

add docs on c7n-left docker image usage and signature validation

### CODE REVIEW

**Review by ajkerrigan — APPROVED** [review]
lgtm for the most part, and having a taverse filter example is cool :+1: . Couple nits/suggestions inline

File: tools/c7n_left/README.md [hunk 1]
```diff
@@ -9,15 +9,31 @@ source asset evaluation.
...
 ```shell
 pip install c7n_left
 ```
 
+We also provide signed docker images. These images are built on top of chainguard's [wolfi linux
+distribution](https://www.chainguard.dev/unchained/introducing-wolfi-the-first-linux-un-distro) which
+is designed to be minimal, auditable, and secure.
+
+```shell
+docker pull c7n_left:dev
```
Comment (ajkerrigan) [inline tools/c7n_left/README.md:24]:
```suggestion
docker pull cloudcustodian/c7n-left:dev
```

File: tools/c7n_left/README.md [hunk 2]
```diff
@@ -9,15 +9,31 @@ source asset evaluation.
...
+```
+
+Images signatures can be verified using [cosign](https://github.com/sigstore/cosign)
+
+```
+set IMAGE (docker image inspect cloudcustodian/c7n-left:dev -f '{{index .RepoDigests 0}}')
```
Comment (ajkerrigan) [inline tools/c7n_left/README.md:30]: Think bash/zsh syntax will be a friendlier default here

```suggestion
export IMAGE=$(docker image inspect cloudcustodian/c7n-left:dev -f '{{index .RepoDigests 0}}')
```

**Review by ajkerrigan — COMMENTED** [review]
Ehhh lost my inline suggestions when I submitted that last review, trying again

File: tools/c7n_left/README.md (hunk 2 above)
Comment (kapilt) [reply tools/c7n_left/README.md]: good call out, thanks

File: tools/c7n_left/README.md (hunk 1 above)
Comment (kapilt) [reply tools/c7n_left/README.md]: nice catch, thanks

**Pull request merged**